### ISSUE TITLE
Implement ML label enrichment for Hansard

### ISSUE THREAD

**Issue by beingkk** [nestauk/discovery_utils#45]
Hey, as a next step after keyword-based labelling, let's add the ML-based labelling to the enrichment pipeline.

- Let's start with adding it to the Hansard data (and then can implement on Crunchbase as a new issue)
- You'll find the inference code in [here](https://github.com/nestauk/discovery_utils/tree/dev/discovery_utils/horizon_scout)
-I think it should overall work similarly to the keyword labelling - input is IDs and text, and output should be labels stored in the 'label store'.
-Note that we should distinguish between running the enrichment only on new data vs running on the whole dataset. I think the weekly dag should only run on the new data - whereas we should be able to run it manually on the full data (eg, need to do it only once, or every time when the model gets updated).

Hope this is clear enough to get started - and please don't hesitate when questions arise!
